Userland/Libraries/LibJS/Runtime/ArrayPrototype.cpp

```diff
@@ -151,8 +151,12 @@ static Object* array_species_create(GlobalObject& global_object, Object& origina
 {
     auto& vm = global_object.vm();
 
-    if (!Value(&original_array).is_array(global_object))
-        return Array::create(global_object, length);
+    if (!Value(&original_array).is_array(global_object)) {
+        auto array = Array::create(global_object, length);
+        if (vm.exception())
+            return {};
+        return array;
```

Comment:
I don't mind these explicit checks at all but I often skip them myself - if we need an exception check here, we need one at the call site as well, and might as well discard the return value there, so technically there should be no difference in behaviour.

But again, this is more explicit and that's good :)